java/client/src/org/openqa/selenium/Capabilities.java

```diff
@@ -86,7 +86,7 @@ default boolean isJavascriptEnabled() {
   default boolean is(String capabilityName) {
     Object cap = getCapability(capabilityName);
     if (cap == null) {
-      // javascriptEnabled is true if not set explicitly
```

Comment:
What is wrong with this message?

Reply:
The fact that it neither starts with a capital letter nor ends with a period.

Thanks for reviewing.